torch/distributed/pipelining/PipelineSchedule.py

```diff
@@ -195,7 +195,15 @@ def _merge_outputs(self, output_chunks: List[Any]) -> Any:
         )
 
 
-def sorted_batch_isend_irecv(p2p_ops: List[dist.P2POp]) -> Dict[int, dist.Work]:
+def batch_p2p(p2p_ops: List[dist.P2POp], desc: Optional[str] = None):
+    desc_str = f"{desc}, " if desc else ""
+    logger.debug(f"batch_p2p {desc_str}{p2p_ops}")  # noqa: G004
```

Comment:
Would print of `p2p_ops` cause a tensor sync like the case of printing loss? I don't know. Just asking wildly here.

Reply:
no, it only prints shape/dtype. no sync.  unless i'm missing something?  (see previous PR)

Reply:
oh oh, I missed that this PR is on a stack. LGTM